tlatools/org.lamport.tlatools/customBuild.xml

```diff
@@ -143,17 +143,53 @@
 	<!-- Compiles the TLA+ tools code -->
 	<target name="compile" depends="clean, generate">
 		<echo>
-			==================================================================
-			= This code uses sun.misc.Unsafe for manual memory management in =
-			= the class src/tlc2/tool/fp/LongArray.java. This unsafe API is  =
-			= marked as internal meaning it can change without warning.      =
-			= Ordinarily this would result in a lot of compiler warnings,    =
-			= but to reduce noise these have been explicitly silenced. If    =
-			= the build fails after upgrading Java, it might be because this =
-			= API has been changed.                                          =
-			==================================================================
+==================================================================
+= This code uses sun.misc.Unsafe for manual memory management in =
+= the class src/tlc2/tool/fp/LongArray.java. This unsafe API is  =
+= marked as internal meaning it can change without warning.      =
+= Ordinarily this would result in a lot of compiler warnings,    =
+= but to reduce noise these have been explicitly silenced. If    =
+= the build fails after upgrading Java, it might be because this =
+= API has been changed.                                          =
+==================================================================
 		</echo>
 		<mkdir dir="${class.dir}" />
+		<!--
+			Separate compilation step for files which must have their warnings
+			uniformly ignored. Java 11 removed the -XDignore.symbol.file compiler
+			parameter that worked in Java 8, so this is the new method. For some
+			reason Java *really, really* does not want you to silence the warnings
```

Comment:
That's because they *really, really* want you to get rid of your use of Unsafe.

Reply:
Per [JEP 471](https://openjdk.org/jeps/471):

>Distinct from deprecation warnings, javac has issued warnings about the use of sun.misc.Unsafe since 2006:

> warning: Unsafe is internal proprietary API and may be removed in a future release

>These warnings will continue to be issued, and **cannot be suppressed**.